src/SourceBuild/content/eng/tools/PrBaselinePublisher/Publisher.cs

```diff
@@ -65,7 +65,9 @@ private record ChangedFile(string Path, string Content);
         }
         else
         {
-            Log.LogError("Invalid pipeline.");
+            string errorMessage = "Invalid pipeline.";
+            Log.LogError(errorMessage);
```

Comment:
I was thinking this could be moved to the main entrypoint of the app so that this is called whenever an exception is thrown.